.travis.yml

```diff
@@ -21,6 +21,15 @@ cache:
     - $HOME/.npm
     - $HOME/.cache # includes bower's cache
 
+env:
+  - NPM_SCRIPT=test-all:cover
+
+matrix:
+  fast_finish: true
```

Comment:
@stefanpenner - When this is enabled, the commit status is updated to failing once the _first_ job fails. Without this option, the commit status stays "pending" until all tests have concluded and is then marked as failing if one of the jobs failed.